### DISCUSSION TITLE
Is there a fsnotify equivalent in the standard lib ?

### DISCUSSION THREAD

**Discussion by olup** [vlang/v#7544] [General]
Is this the place to ask dumb question ? I couldn't find a function to register a callback on file system change (file change, moved, etc...) Can maybe someone point me towards it ?

**Comment by spytheman** [comment]
There is not.

**Comment by medvednikov** [comment]
It's planned.

**Reply by olup** [reply]
Cool ! I guess it would use inotify on Linux and equivalent for macos and windows. I am trying to figure out how to do it as an exercise to learn v a bit ( most exciting thing I have come across in a long time). So for inotify would you import the inotify C libs the way it's done in the sqlite C example ?
